### PR TITLE
Refactor 3rd party scripts

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Every 3rd party module is basically doing the same thing many times, so I figured we could do that same thing only once instead.

cc @guardian/commercial-dev

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/third-party-tags.js [hunk 1]
```diff
@@ -84,12 +77,30 @@ define([
...
     function loadOther() {
-        imrWorldwide.load();
-        remarketing.load();
+        var services = [
+            audienceSciencePql,
+            audienceScienceGateway,
+            imrWorldwide,
+            remarketing,
+            krux
+        ].filter(function (_) { return _.shouldRun; });
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags.js]: I think we should use fully descriptive variables where possible – where here, `_` is really `service`?

Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags.js]: Generally I agree, it is especially important when going through long procedural processes. But in this case, I feel like the important information `shouldRun` should be the focus of this step of the process (the filtering) and we don't really care about what should run.

File: static/src/javascripts/projects/common/modules/commercial/third-party-tags/audience-science-pql.js [hunk 2]
```diff
@@ -1,71 +1,72 @@
...
-                        return placement[1]['default']; //eslint-disable-line
-                    }).and(map, function (placement) {
-                        return ['pq_' + placement[0], 'T'];
-                    }).and(zipObject).valueOf();
-                // set up the global asiPlacements var in case dfp returns before asg
-                window.asiPlacements = storedSegments[section];
-            }
-            return segments;
-        };
+    function onLoad() {
```
Comment (JonNorman) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags/audience-science-pql.js]: What are we actually doing here/ why? Can pop over and discuss if it is easier.

**Comment by JonNorman** [comment]
Apart from the slightly confusing (to me at least) flow of `audience-science-pql`, looks good to me :+1:

File: static/src/javascripts/projects/common/modules/commercial/third-party-tags/audience-science-pql.js (hunk 2 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags/audience-science-pql.js]: TBH I don't know, seems we swap two values: one we receive from the 3rd party, one we store in local storage.

Comment (JonNorman) [inline static/src/javascripts/projects/common/modules/commercial/third-party-tags/audience-science-pql.js]: 😂  it makes none of the sense.

**Pull request merged**